tensorflow/python/layers/convolutional.py

```diff
@@ -290,7 +295,14 @@ def __init__(self, filters,
                trainable=True,
                name=None,
                **kwargs):
-    super(Convolution1D, self).__init__(
+    if padding == "causal":
```

Comment:
@szpssky note the code block is before invoking `super(Conv1D, self).__init__(`.  In your case, `causal` leaks into its parent class. I think there might be two reasons:
1. You might paste the code into wrong place by mistake, or
2. The multiple inheritance order of python varies on different environment, or I misunderstand the rule.